src/pages/Main/index.js

```diff
@@ -137,6 +211,59 @@ let Main = ({ searchTerm, fetchResults, push }) => {
           </div>
         </section>
       )}
+
+      <section className="main__section main__section--explore">
+        <div className="main__container">
+          <div className="main__explore-card-container">
+            <div
+              className="main__explore-card"
+              style={{
+                backgroundImage: `url(${networkBottomIcon})`,
+                backgroundSize: '80%'
+              }}
+            >
+              <div className="main__explore-card__title">Species Compendia</div>
+              <div className="main__explore-card__body">
+                Our Species compendia offers datasets which has been processed
```

Comment:
* Is `S` in `Species` supposed to be lower case?
* `which has` probably should be `which have` or `that have`.

Reply:
@dongbohu thanks a lot for reviewing this!! I guess that sounds better, let's confirm with @dvenprasad. 

Deepa, do you agree? We're referring to [this block](https://projects.invisionapp.com/d/main#/console/13557531/300190943/preview)?

![image](https://user-images.githubusercontent.com/1882507/57396852-561a2600-7199-11e9-8bfb-960043d10498.png)

Reply:
- Yes `which have` sounds better.
- Capitalize `C` as in `Our Species Compendia` as well since we are talking about an offering.

Reply:
:+1:

Reply:
Thanks, sure!

```suggestion
                Our Species Compendia offers datasets which have been processed
```